rcutils/logging.py

```diff
@@ -26,10 +26,11 @@
     ('name', 'The name of the logger'),
 ))
 name_args = {'name': 'name'}
+name_doc_lines = []
```

Comment:
Is the assumption here that `name` is explicit enough and doesnt need a doc string? Or will this get a message added in a follow-up PR ?

Reply:
the "doc lines" are not for documenting the parameters (that's line 26), it's for what comes after "Log a message with severity DEBUG with the following conditions". name isn't a condition in that sense so it doesn't (and probably won't ever) have a doc line in that sense